bibcheck/plugins/convert_grid_to_ror.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2013, 2019 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+"""Bibcheck plugin to convert institution IDs from GRID to ROR."""
+
+import json
+
+GRID_TO_ROR_PATH = '/opt/cds-invenio/var/data/grid_to_ror.json'
+
+with open(GRID_TO_ROR_PATH) as f:
+    grid_to_ror = json.load(f)
+
+def check_record(record):
+    for pos, grid in record.iterfield('035__a', ('9', 'GRID')):
```

Comment:
W: 29, 8: Unused variable 'pos' (unused-variable)

for clarity you could replace `pos` with `_`